platform/linux-generic/arch/aarch64/odp_crypto_armv8.c

```diff
@@ -405,9 +434,9 @@ static int process_aes_gcm_param(odp_crypto_generic_session_t *session)
 
 	/* Set function */
 	if (ODP_CRYPTO_OP_ENCODE == session->p.op)
-		session->cipher.func = aes_gcm_encrypt;
+		session->func = aes_gcm_encrypt;
```

Comment:
Alternatively:
```c
session->func = ODP_CRYPTO_OP_ENCODE == session->p.op ? aes_gcm_encrypt : aes_gcm_decrypt;
```

Reply:
I do not see the benefit of that here. I think it it would be slightly less readable and would also be a slightly bigger change.